### ISSUE TITLE
App Notifications

### ISSUE THREAD

**Issue by SILVERWAVE20** [Vencord/plugin-requests#22]
### Plugin Description

Mainly, a PC version of mobile's notifications, where a DM will pop up for a limited time somewhere on discord itself, when not clicked in that chat

### Existing Plugin for other mod

https://betterdiscord.app/plugin/AppNotifications

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Oi-jojo** [comment]
It alrdy exists, here it is. https://github.com/ethan-davies/ToastNotificationsMerge/tree/master. You need dev shell for this tho

**Comment by ethan-davies** [comment]
Yes, also that could potentially be merged if we get further response on:
- https://github.com/Vendicated/Vencord/pull/1806
